src/datasource/datasource.js

```diff
@@ -282,6 +285,72 @@ export class DataSource {
     return rawTxResponse;
   }
 
+  async sendTransactionFromLedger(publicKey, signature, tx) {
+    const {
+      chain_id: chainID,
+      entropy,
+      fee,
+      memo,
+      msg: {
+        value: { amount, from_address: fromAddress, to_address: toAddress },
+      },
+    } = tx;
+
+    const txSignature = new TxSignature(
+      Buffer.from(publicKey, "hex"),
+      Buffer.from(signature, "hex")
+    );
+
+    const transactionSender = new TransactionSender(
+      this.__pocket,
+      null,
+      null,
+      true
+    );
+
+    const itxSender = transactionSender.send(fromAddress, toAddress, amount);
+
+    const unsignedTransaction = itxSender.createUnsignedTransaction(
+      chainID,
+      fee[0].amount,
```

Comment:
This can break if `fee` is undefined or empty.

Reply:
Not really, this fee comes from the transportContext and we are making sure the fee exists so it will always have a value.